tests/common/plugins/loganalyzer/__init__.py

```diff
@@ -49,7 +50,18 @@ def analyze_logs(analyzers, markers, node=None, results=None, fail_test=True, st
 def loganalyzer(duthosts, request):
     if request.config.getoption("--disable_loganalyzer") or "disable_loganalyzer" in request.keywords:
         logging.info("Log analyzer is disabled")
+        mark_file = "/etc/sonic/disable_loganalyzer_mark"
+        for duthost in duthosts:
+            try:
+                duthost.shell("touch %s" % mark_file, module_ignore_errors=True)
```

Comment:
>touch [](http://example.com/codeflow?start=31&length=5)

I could not find any code related the usage of the file "disable_loganalyzer_mark". So I could not really understand "What is the motivation for this PR?"

Reply:
https://github.com/sonic-net/sonic-utilities/pull/3102
db migrator will use this flag to detect loganalyzer is disabled in end to end test.

Reply:
Updated with simplified design.